jose.pyx

```diff
@@ -0,0 +1,141 @@
+from libc.stdlib cimport free
+cimport jansson
+cimport jose
+
+import json
+
+def jwk_generate(jwk):
+    cdef jansson.json_t *cjwk = NULL
+    cdef char *ret = NULL
+
+    assert isinstance(jwk, dict)
+
+    try:
+        cjwk = jansson.json_loads(json.dumps(jwk).encode(u"UTF-8"), 0, NULL)
```

Comment:
PR #2 implements Python to Jansson json_t. May I suggest that we merge my PR first so you can use the feature in your PR?